UI.PageModels/Dtos/ScreeningEntityDto.cs

```diff
@@ -0,0 +1,11 @@
+namespace UI.PageModels.Dtos;
+
+public class ScreeningEntityDto
```

Comment:
Have you considered using a record type, you can achieve something similar with less code, especially since this data is not likely to chance once you've defined it

Reply:
good shout, thanks. Have updated it